File: LeetCode/1-100/75. Sort Colors/src/Solution.java
import java.util.Arrays;

class Solution {
    public void sortColors(int[] nums) {
        int[] colors = new int[3];
        for (int i = 0; i < nums.length; i++) {
            colors[nums[i]]++;
        }
        colors[1] += colors[0];
        colors[2] += colors[1];
        Arrays.fill(nums, 0, colors[0], 0);
        Arrays.fill(nums, colors[0], colors[1], 1);
        Arrays.fill(nums, colors[1], colors[2], 2);
    }
}

File: LeetCode/1-100/75. Sort Colors/Solution.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int counter[3] = {0, 0, 0};
        for (int curr: nums) {
            counter[curr]++;
        }
        fill(nums.begin(), nums.begin() + counter[0], 0);
        fill(nums.begin() + counter[0], nums.begin() + counter[0] + counter[1], 1);
        fill(nums.begin() + counter[0] + counter[1], nums.end(), 2);
    }
};